frontend/src/components/Card.tsx

```diff
@@ -108,102 +89,69 @@ const Card = ({
       )}
       {/* Render project summary using Markdown */}
       <Markdown content={summary} className="py-2 pr-4 text-gray-600 dark:text-gray-300" />
-      {/* Display leaders of the project */}
-      {leaders && (
-        <h2 className="py-1">
-          <span className="font-semibold text-gray-600 dark:text-gray-300">
-            {leaders.length > 1 ? 'Leaders: ' : 'Leader: '}
-          </span>
-          {leaders.map((leader, index) => (
-            <span key={`${leader}-${index}`} className="text-gray-600 dark:text-gray-300">
-              {index !== leaders.length - 1 ? `${leader}, ` : leader}
-            </span>
-          ))}
-        </h2>
-      )}
-      <div className="flex w-full flex-col gap-4 pr-4">
+
+      <div
+        className={
+          social && social.length > 0
+            ? 'flex w-full flex-col gap-4 pr-4'
```

Comment:
Can we make this gap smaller? That's way too much space 
```suggestion
            ? 'flex w-full flex-col gap-2 pr-4'
```